posthog/test/base.py

```diff
@@ -105,6 +105,166 @@
 unittest.util._MAX_LENGTH = 2000  # type: ignore
 
 
+def clean_varying_query_parts(query, replace_all_numbers):
+    # :TRICKY: team_id changes every test, avoid it messing with snapshots.
+    if replace_all_numbers:
+        query = re.sub(r"(\"?) = \d+", r"\1 = 2", query)
+        query = re.sub(r"(\"?) IN \(\d+(, ?\d+)*\)", r"\1 IN (1, 2, 3, 4, 5 /* ... */)", query)
+        query = re.sub(r"(\"?) IN \[\d+(, ?\d+)*\]", r"\1 IN [1, 2, 3, 4, 5 /* ... */]", query)
+        # replace "uuid" IN ('00000000-0000-4000-8000-000000000001'::uuid) effectively:
+        query = re.sub(
+            r"\"uuid\" IN \('[0-9a-f-]{36}'(::uuid)?(, '[0-9a-f-]{36}'(::uuid)?)*\)",
+            r""""uuid" IN ('00000000-0000-0000-0000-000000000000'::uuid, '00000000-0000-0000-0000-000000000001'::uuid /* ... */)\n""",
+            query,
+        )
+
+    else:
+        query = re.sub(r"(team|cohort)_id(\"?) = \d+", r"\1_id\2 = 2", query)
+        query = re.sub(r"\d+ as (team|cohort)_id(\"?)", r"2 as \1_id\2", query)
```

Comment:
The only thing that feels weird here is that this replacement dummy value `2` look just like normal one 😅 whereas `0000000-0000-0000-0000-000000000000` seems obviously like a test placeholder. Something like `99999` would be more obvious (and `2` has been here before this PR, so I'm introducing scope creep)

Reply:
only a tiny bit of creep though

your wish is my command

Reply:
🤣 111 files changed 📈